api/contrib/envoy/extensions/filters/network/postgres_proxy/v3alpha/postgres_proxy.proto

```diff
@@ -55,6 +55,8 @@ message PostgresProxy {
   // Postgres filter is able to read Postgres payload in clear-text. It happens when
   // a client established a clear-text connection to Envoy or when a client established
   // SSL connection to Envoy and Postgres filter is configured to terminate SSL.
+  // In order for upstream encryption to work, the corresponding cluster must be configured to use
```

Comment:
flagging that this wont render as a newline - not sure it that was your intention

https://storage.googleapis.com/envoy-pr/b047a5f/docs/api-v3/extensions/filters/network/postgres_proxy/v3alpha/postgres_proxy.proto.html#extensions-filters-network-postgres-proxy-v3alpha-postgresproxy

Reply:
I think it's fine as is.